example/compose/authelia/Dockerfile.backend

```diff
@@ -1,6 +1,9 @@
 FROM golang:1.13-alpine
 
-RUN apk --no-cache add gcc musl-dev
+#TODO(nightah): Remove when turning off Travis
+ARG TRAVIS
+RUN if [ "$TRAVIS" == "true" ]; then apk --no-cache add gcc musl-dev; fi
```

Comment:
Why don't we need to install gcc and the libc in buildkite setup?

Reply:
Because we aren't building the binaries again in Buildkite, they're shared from the build step to speed up the tests.

https://github.com/authelia/authelia/blob/37c6c4383e18af2894b3c4e4dc6e8c4bcda0885b/.buildkite/pipeline.yml#L6-L8

Reply:
Ah yes, got it!